steps/pebble-build/1.0.0/step.yml

```diff
@@ -0,0 +1,75 @@
+title: Pebble build
+summary: Builds a Pebble project
+description: |
+  Builds a [Pebble project](https://developer.pebble.com) after installing SDK (if necessary).
+website: https://github.com/HBehrens/bitrise-step-pebble-build
+source_code_url: https://github.com/HBehrens/bitrise-step-pebble-build
+support_url: https://github.com/HBehrens/bitrise-step-pebble-build/issues
+published_at: 2016-07-01T17:33:30.501763379-07:00
+source:
+  git: https://github.com/HBehrens/bitrise-step-pebble-build.git
+  commit: dac55032dc4ac855645dd6c27b0910fcaa413119
+host_os_tags:
+- osx-10.10
+- osx-10.11
+- osx-10.12
+is_requires_admin_user: false
+is_always_run: false
+inputs:
+- opts:
+    description: |-
+      Specify the path to your Pebble project.
+      Use an empty value to skip building the project and only install the SDK.
+    title: Pebble project path
+  project_path: $BITRISE_SOURCE_DIR
+- opts:
+    description: |-
+      Speed up Pebble SDK installation by skiping the emulator.
+      If you only build your projects, it's safe to do this.
+    title: Install SDK without Emulator?
+    value_options:
+    - "yes"
+    - "no"
+  sdk_without_emulator: "yes"
+- opts:
+    description: |-
+      Speed up Pebble SDK installation by skiping Freetype.
+      Only if you don't use any custom fonts, it's safe to set this to "yes".
+    title: Install SDK without FreeType?
+    value_options:
+    - "yes"
+    - "no"
+  sdk_without_freetype: "no"
+- opts:
+    description: |-
+      Speed up Pebble SDK installation by skiping Node.
+      Only if you don't use Pebble Packages, it's safe to set this to "yes".
+    title: Install SDK without Pebble Packages?
+    value_options:
+    - "yes"
+    - "no"
+  sdk_without_node: "no"
+- opts:
+    description: |-
+      If set to "yes" (recommended), Pebble collects metrics on your usage of their developer tools.
+      This helps them to help prioritise further development of their tooling.
+    title: Send analytics about SDK usage to Pebble?
+    value_options:
+    - "yes"
+    - "no"
+  sdk_enable_analytics: "yes"
+- opts:
+    description: |-
+      To use the Pebble SDK, you must agree to the following:
+
+      PEBBLE TERMS OF USE
+      https://developer.getpebble.com/legal/terms-of-use
+
+      PEBBLE DEVELOPER LICENSE
+      https://developer.getpebble.com/legal/sdk-license
+    is_expand: true
+    title: Accept Pebble TERMS OF USE and PEBBLE DEVELOPER LICENSE?
+    value_options:
+    - "yes"
+    - "no"
+  sdk_accept_legalese: "no"
```

Comment:
What do you think about this: 
- remove the value options
- leave the default value empty
- mark the input as required (`is_required: true`)
- complete the description with something like: __You have to provide `yes` as input value to accpect the licence.__

The only problem with default `false` is that users will not read the whole step description, and probably will skipp to set this input and build will fail (hopefully just once).

If you follow my mentions the bitrise UI will mark the missing input and probably this way more users will provide the desired input at first.